### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by cn0811** [share121/inter-knot#4592] [绝区零]
6+5安比和0+1狼哥怎么取舍，有青衣

**Comment by shidijun** [comment]
说实话建议都用上 可以两队分别一个<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" > （我狼叔没专武 安比才5+3 你知道你多让人嫉妒吗<img src="https://bbs-static.miyoushe.com/static/2024/03/21/8fe47df50ba7e17e012e8ba5baaaad49_4838310462379389617.png" alt="丽都漫步-安东" width="75" >）

**Reply by cn0811** [reply]
但是我还有一个0+1的青衣和0+1的珂蕾妲，怎么取舍

**Reply by cn0811** [reply]
珂蕾妲不想用，这三个不知道怎么取舍

**Comment by LMA936** [comment]
灵活选人（练度的话建议优先青衣，最泛用，其次狼哥，无敌帧长）
首先看队伍适配程度（适合触发自己和队友的额外能力，以及一些异常积蓄需求）
其次看敌人主要弱点和抗性（会影响失衡值的）
都一样就无脑青衣（
<img src="https://bbs-static.miyoushe.com/static/2024/09/04/f2ca6cb970baa36ebb1b8950f3894fb2_4590307880523876022.png" alt="玛拉妮-慌张" width="75" >
